### DISCUSSION TITLE
Where can I get the Apple Music token?

### DISCUSSION THREAD

**Discussion by FloJam2005** [TopiSenpai/LavaSrc#46] [Q&A] [answered]
Hey,
Where can I find the "mediaAPIToken" required for constructing the AppleMusicSourceManager?

**Comment by topi314** [comment, marked as answer]
This argument is optional. You can leave it empty and it will try to extract one from `music.apple.com` 
If that fails you need to extract that yourself. It's hidden in some js files on their site

**Reply by FloJam2005** [reply]
all right, thank you very much :)
